### PR TITLE
Add support custom max value for barcharts

### PR DESCRIPTION
**Pull request by rifatx**
With this PR, it will be possible to set a custom max value for barcharts, so that multiple charts could be visually aligned.

This PR closes issue #524.

### CODE REVIEW

File: src/Spectre.Console/Widgets/Charts/BarChart.cs [hunk 1]
```diff
@@ -43,6 +43,12 @@ public sealed class BarChart : Renderable, IHasCulture
         /// <remarks>Defaults to invariant culture.</remarks>
         public CultureInfo? Culture { get; set; }
 
+        /// <summary>
+        /// Gets or sets the fixed max value for a bar chart.
+        /// </summary>
+        /// <remarks>Defaults to 0, which corresponds to largest value in chart.</remarks>
+        public double MaxValue { get; set; } = 0;
```
Comment (patriksvensson) [inline src/Spectre.Console/Widgets/Charts/BarChart.cs:50]: Perhaps this could be nullable to indicate that no explicit max value has been set?

**Review by patriksvensson — CHANGES_REQUESTED** [review]
LGTM apart from a minor thing 👍

File: src/Spectre.Console/Widgets/Charts/BarChart.cs (hunk 1 above)
Comment (rifatx) [reply src/Spectre.Console/Widgets/Charts/BarChart.cs]: makes sense 👍 updated the PR.

File: src/Spectre.Console/Extensions/BarChartExtensions.cs [hunk 2]
```diff
@@ -238,5 +238,22 @@ public static BarChart RightAlignLabel(this BarChart chart)
...
             return chart;
         }
+
+        /// <summary>
+        /// Sets the max fixed value for the chart.
+        /// </summary>
+        /// <param name="chart">The bar chart.</param>
+        /// <param name="maxValue">Max value for the chart.</param>
+        /// <returns>The same instance so that multiple calls can be chained.</returns>
+        public static BarChart WithFixedMaxValue(this BarChart chart, double maxValue)
```
Comment (patriksvensson) [inline src/Spectre.Console/Extensions/BarChartExtensions.cs:248]: Could we rename this to `WithMaxValue` instead?

**Review by patriksvensson — CHANGES_REQUESTED** [review]

File: src/Spectre.Console/Extensions/BarChartExtensions.cs (hunk 2 above)
Comment (rifatx) [reply src/Spectre.Console/Extensions/BarChartExtensions.cs]: 👍

**Review by patriksvensson — APPROVED** [review]
LGTM!

**Review by patriksvensson — APPROVED** [review]

**Comment by patriksvensson** [comment]
Merged! Thank you for your contribution. Much appreciated! 👍

**Pull request merged**